string env_name
string name_file
---
bool finished